File: TriviaServer/TriviaServer/SubmitAnswerResponse.cpp
#include "SubmitAnswerResponse.h"

SubmitAnswerResponse::SubmitAnswerResponse(int status)
{
	this->messageCode = 83;
	this->_status = status;
	this->_correctAnswerId = 0; //ID of correct answer is always 0
}

nlohmann::json SubmitAnswerResponse::castToJson() const
{
	return nlohmann::json{ {"status" , _status}, {"correctAnswerId", this->_correctAnswerId} };
}

File: TriviaWPFClient/TriviaClient/SubmitAnswerResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TriviaClient
{
    class SubmitAnswerResponse : Response
    {
        public const int MESSAGE_CODE = 83;
        public int status { get; set; }
        public string correctAnswerId { get; set; }
        public SubmitAnswerResponse() : base(MESSAGE_CODE) { }

    }
}
